modules/virt-creating-configmap.adoc

```diff
@@ -0,0 +1,264 @@
+// Module included in the following assemblies:
+// * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
+[id='virt-creating-configmap_{context}']
+= Creating a custom ConfigMap for importing a virtual machine
+
+You can check whether the operating system of the Red Hat Virtualization (RHV) virtual machine that you are importing maps to a {product-title} common template.
+
+If the RHV VM's operating system does not have a corresponding {product-title} template, you must create a custom ConfigMap to map the VM's operating system to a common {product-title} template. The template is used to create the new virtual machine in {VirtProductName}.
+
+[IMPORTANT]
+====
+In the current release, you must create a Windows template and a custom ConfigMap if you import a Windows VM. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1856654[BZ1856654] for details.
+====
+
+.Procedure
+
+. Locate your RHV operating system designation in the `/etc/ovirt-engine/osinfo.conf.d/00-defaults.properties` file on the Manager machine, for example, `rhel_6_ppc64`:
++
+----
+# rhel6ppc64(1003, OsType.Linux, true),
+os.rhel_6_ppc64.id.value = 1003
+os.rhel_6_ppc64.name.value = Red Hat Enterprise Linux up to 6.8
+os.rhel_6_ppc64.cpu.unsupported.value.4.2 = power8, power9
+os.rhel_6_ppc64.cpu.unsupported.value.4.3 = power8, power9
+os.rhel_6_ppc64.derivedFrom.value = other_linux_ppc64
+----
+
+. Check the list of common operating system templates installed on the {product-title} cluster by running the following command:
++
+----
+$ oc get templates -n openshift --show-labels | tr ',' '\n' | grep os.template.kubevirt.io | sed -r 's#os.template.kubevirt.io/(.*)=.*#\1#g' | sort -u
+----
++
+If your RHV VM operating system does not have a corresponding {product-title} template, you must create a custom ConfigMap. If you are importing a VM with a Windows operating system, you must create a Windows template and a custom ConfigMap.
+
+. If you are importing a VM with a Windows operating system, create a Windows {product-title} template:
```

Comment:
this windows custom CNV (- I think) template

Reply:
OK